base/math.jl

```diff
@@ -74,15 +74,15 @@ macro evalpoly(z, p...)
         ai = symbol("a", i)
         push!(as, :($ai = $a))
         a = :(muladd(r, $ai, $b))
-        b = :(muladd(-s, $ai, $(esc(p[i]))))
+        b = :($(esc(p[i])) - s * $ai) # don't have efficient multiply-subtract
```

Comment:
The best thing would be to file an issue specifically for getting access to fast multiply-subtract (e.g. by tweaking compiler flags), and cite that issue here.